translator2024/CanApiDraft.hpp

```diff
@@ -0,0 +1,83 @@
+//Draft of the format of what will be the generated API header
+#ifndef CanApiv04_INCLUDE_GUARD
+#define CanApiv04_INCLUDE_GUARD
+#include "mcp2515.h" //for certain datatypes used in this file
+
+namespace CANHelper::Messages //union containing all messages is at bottom
+{
+#ifdef USE_MSG_Telemetry_TimeAndFix
+#define CAN_ID_Telemetry_TimeAndFix 0x0F6
+#define CAN_DLC_Telemetry_TimeAndFix 8
+	struct Telemetry::_TimeAndFix { //Python code will remove underscores from item names (just havnt changed them here)
```

Comment:
why the comment instead of removing the underscore here?

Reply:
resolved with https://github.com/DUEM/CANTranslator/pull/10/commits/7991e6e98b24f421fc604c4e7814634c0dc08116